bcl2fastq/index_summary.py

```diff
@@ -297,5 +211,49 @@ def update_pie_chart(run_alias, known_json, unknown_json):
     )
 
 
+@app.callback(
+    [dep.Output('run_select', 'value'),
+     dep.Output('error', 'displayed')],
+    [dep.Input('url', 'pathname')]
+)
+def change_url(value):
+    """ Allows user to enter Run name in URL which will update dropdown automatically, and the graphs.
+        If User enters any value that's not a valid run an error box will pop up and return user to most recent run
+
+        Parameters:
+             value: the URL input after the port (the pathname).
```

Comment:
`the URL input after the port` -> `the user-requested path` ? (We could host the website so the port isn't displayed in the URL, but this function would still work.
A quick note about naming: it's preferable to have more specific names (for instance, `path` rather than `value`). This helps the reader since they don't have to remember what the more general name refers to.